go/go.mod

```diff
@@ -1,9 +1,17 @@
 module github.com/cncf/xds/go
 
-go 1.11
+go 1.20
```

Comment:
I think I was wrong to bump it that high in the first place. This is the minimal version of go required to use this module: https://go.dev/ref/mod#go-mod-file-go

So projects like grpc won't be able to use this package because they declare lower minimal version, f.e. 1.19 in gRPC's case: https://github.com/grpc/grpc-go/blob/287c47355e154ac2193bf743c886656cd48e753c/go.mod#L3

I think we should set it to something reasonable, higher than 1.11, but lower than 1.20.

Reply:
Should this be set to the version of grpc (1.19) then?